### PR TITLE
Add drift remediation configuration document

### PR DESCRIPTION
**Pull request by RedbackThomson**
Issue #, if available:
Closes https://github.com/aws-controllers-k8s/community/issues/1810

Description of changes:
Adds a new document to the "Getting Started" tab of the documentation which outlines the definition and configuration options for ACK drift remediation. Also adds a link to it from the "How it Works" page.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/content/docs/community/how-it-works.md [hunk 1]
```diff
@@ -61,6 +61,9 @@ an ACK service controller will look for any drift and attempt to remediate. As
 part of the remediation, an ACK service controller will reconfigure the managed
 resource based on the `Spec`.
 
+For more information about configuring the drift remediation period, see
```
Comment (jljaco) [inline docs/content/docs/community/how-it-works.md:64]: Can you please modify the sentence above to read `...every 10 hours (unless a different frequency is configured)...`?  It's great to point the user to more info, but folks who read only this page might come away with the impression that it's always 10 hours.

File: docs/content/docs/user-docs/drift-recovery.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+toc: true
+---
+
+Kubernetes controllers work on the principal of [constant
+reconciliation][constant-reconciliation]. In essence, they continuously look at
+the current desired state of the system and compare it to the actual state,
+using the difference to determine the action required to get to the desired end
+result.
+
+Once a controller has reconciled a resource to it's desired state, the
```
Comment (jljaco) [inline docs/content/docs/user-docs/drift-recovery.md:19]: spelling nit: `it's` -> `its`

File: docs/content/docs/user-docs/drift-recovery.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+controller shouldn't need to continue reconciling - the actual state of the
+resource meets the specification. However, this is only true for closed systems,
+where the controller is the only actor interacting with a resource.
+Unfortunately, ACK controllers don't act in a closed system. ACK controllers are
+not the only actor capable of modifying the actual state of any AWS resources -
+other programs, or even people, may have their own privileges. When another
+actor modifies a resource after the ACK controller has reconciled it to its
+desired state, that's called "Drift".
```
Comment (jljaco) [inline docs/content/docs/user-docs/drift-recovery.md:27]: spelling nit: don't capitalize "drift"

File: docs/content/docs/user-docs/drift-recovery.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+All override periods are logged to stdout when the controller is started.
+{{% /hint %}}
+
+## Overriding the drift remediation period
+
+### For all resources owned by a controller
+
+If you would like to decrease the drift remediation period for *all* resources
+owned by a controller, update the `reconcile.defaultResyncPeriod` value in the
+`values.yaml` file with the number of seconds for the new period, like so:
```
Comment (jljaco) [inline docs/content/docs/user-docs/drift-recovery.md:54]: nit: `Helm chart 'values.yaml' file`

**Review by jljaco — APPROVED** [review]
Looks good to me!  Left a few spelling/language nits, totally non-blocking.

Leaving this for @A-Hilaly for final approval if he has time :)

**Review by A-Hilaly — APPROVED** [review]
Nice!! Thank you @RedbackThomson 
/lgtm

**Pull request merged**